tree/ntuple/v7/src/RField.cxx

```diff
@@ -540,6 +542,24 @@ void ROOT::Experimental::RField<ROOT::Experimental::ClusterSize_t>::AcceptVisito
 
 //------------------------------------------------------------------------------
 
+void ROOT::Experimental::RField<ROOT::Experimental::RNTupleCardinality>::GenerateColumnsImpl(
+   const RNTupleDescriptor &desc)
+{
+   EnsureColumnType({EColumnType::kIndex}, 0, desc);
+   RColumnModel model(EColumnType::kIndex, true /* isSorted*/);
```

Comment:
Why is it tagged as a sorted column?

Reply:
Because the column stores offsets, which are by definition sorted.

Reply:
Oh, right!